tpr/backoff.go

```diff
@@ -0,0 +1,25 @@
+package tpr
+
+import (
+	"time"
+
+	"github.com/cenkalti/backoff"
+)
+
```

Comment:
My intention here is to stop and fail fast if TPR isn't created for whatever reason.

With defaults:

```
tprInitMaxRetries    = 30
tprInitRetryInterval = 3 * time.Second
```

If after ~90 seconds TPR isn't yet initialised we can assume something went wrong and operator should stop. There is no reason to run operator pod which waits forever for TPR to initialise.
WDYT?